### PR TITLE
Update11

### PR DESCRIPTION
**Pull request by mhdostal**
This merges the changes for v100.11.0 from the `update11` branch into `master`.  All commits to `update` have already been reviewed.

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
```
Comment (philium) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:6]: This file is now [ignored by default](https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/InitPackage.swift#L263) when creating a Swift package with newer versions of SPM, so we might want to go ahead and remove it and add it to the gitignore file.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ 380 New York Street
+ Redlands, California, USA 92373
+
+ email: contracts@esri.com
+ */
+
+import PackageDescription
+
+let package = Package(
+    name: "ArcGISToolkit",
```
Comment (philium) [inline Package.swift:28]: Should the name of the package be `arcgis-runtime-toolkit-ios`, to match the name of this repo? Often times that is the case, that the name of the package and the name of the repo containing the package are the same. The library target could still be named `ArcGISToolkit`. Just a thought.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # ArcGIS Runtime Toolkit for iOS
 
-[![doc](https://img.shields.io/badge/Doc-purple)](Documentation) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage) [![CocoaPods](https://img.shields.io/cocoapods/v/ArcGIS-Runtime-Toolkit-iOS)](https://cocoapods.org/)
+[![doc](https://img.shields.io/badge/Doc-purple)](Documentation) [![CocoaPods](https://img.shields.io/cocoapods/v/ArcGIS-Runtime-Toolkit-iOS)](https://cocoapods.org/) [![SPM](https://img.shields.io/badge/SPM-compatible-4BC51D.svg?style=flat)](https://github.com/apple/swift-package-manager/)
```
Comment (philium) [inline README.md:3]: Cool! Did not know there was a badge for this.

File: README.md [hunk 4]
```diff
@@ -24,13 +24,24 @@ To use Toolkit in your project:
...
+### Swift Package Manager
+
+ 1. Open your project in Xcode
+ 2. Go to *File* > *Swift Packages* > *Add Package Dependency* option 
+ 3. Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL
+ 4. Choose version 100.11.0 or a later version. Click Next. Only version 100.11.0 or newer supports Swift Package Manager.
+ 
+ Note: The Toolkit Swift Package adds the ArcGIS SDK Swift Package as a dependency so no need to add both separately. If you already have the ArcGIS SDK Swift Package delete that and just add the Toolkit Swift Package. 
+
+ New to Swift Package Manager? Visit [swift.org/package-manager/](https://swift.org/package-manager/)
```
Comment (philium) [inline README.md:43]: This is very minor, but there appears to be a missing period here.

File: README.md [hunk 5]
```diff
@@ -39,26 +50,14 @@ The *ArcGIS Runtime Toolkit for iOS* has a *Target SDK* version of *12.0*, meani
...
- 2. Clone or download this repo
- 3. Drag and Drop the `Toolkit/ArcGISToolkit.xcodeproj` file into your project through the XCode Project Navigator pane
- 4. Drag the `ArcGISToolkit.framework` from the `ArcGISToolkit.xcodeproj/ArcGISToolkit/Products` folder to the "TARGETS" settings for your application and drop it in the "Embedded Binaries" section in the "General" tab
- 5. Add `import ArcGISToolkit` in your source code and start using the toolkit components 
+ 1. Clone or download this repo
+ 2. Drag and Drop the `arcgis-runtime-toolkit-ios` folder into your project through the Xcode Project Navigator pane
+ 3. Add the *ArcGISToolkit* library in your app, by adding it to the Frameworks, Libraries, and Embedded Content section of the General pane for your app target. The *ArcGISToolkit* library contains the *ArcGIS Runtime SDK for iOS* library, so you don't need to add that separately.
+ 4. Add `import ArcGIS` and `import ArcGISToolkit` in your source code and start using the toolkit components 
+
+Note: Support for Carthage as been dropped for v100.11.0.
```
Comment (philium) [inline README.md:60]: I think this should say "has been dropped".

File: Package.swift (hunk 2 above)
Comment (yo1995) [reply Package.swift:28]: Fixed in #148

**Review by dg0yal — APPROVED** [review]

**Review by philium — APPROVED** [review]

**Review by zkline101 — APPROVED** [review]

**Pull request merged**